### PR TITLE
fix(docs): warn about a potential false positive of Windows app as virus

### LINKED ISSUE #140: [BUG] .exe causes virus alert
**Describe the bug**
The current exe release for Windows. Downloaded from
https://github.com/totalopenstation/totalopenstation/releases/download/v0.5.2/totalopenstation_x64_0.5.2.exe
causes some Antivirus programs to deny downloading because of an alert.

**To Reproduce**
Download on a Windows10 PV with for e.g. Avira activated or upload exe to virustotal which also shows 3 positives
https://www.virustotal.com/gui/file/60c1e2b181fb85238423f033d7645e618c5ad64e654df83a16ffa05509564e14/detection

**Expected behavior**
No alert, download and installable exe on Windoes

### PR DESCRIPTION
**Pull request by steko**
fix #140

### CODE REVIEW

File: docs/installing.rst [hunk 1]
```diff
@@ -71,6 +71,17 @@ included in the executable, without need to install.
 To upgrade to a newer version, just go to the Github download page again.
 No data will be lost!
 
+.. warning::
+   
+   In some cases, there may be a warning about a potential virus
+   threat in the downloaded ``.exe`` file. This is a false positive
+   and we are actively reaching out to antivirus vendors to make sure
+   that Total Open Station is recnognized as genuine software. On a
```
Comment (psolyca) [inline docs/installing.rst]: recognized and not recnognized

I would have write : 

```
[...] as genuine sotware. See `issue #140 `_ for more details. On a technical level [...] open source code. See Github Action workflow `pyinstaller.yml`_.

.. _`pyinstall.yml`: https://github.com/totalopenstation/totalopenstation/actions/workflows/pyinstaller.yml)
```

**Review by psolyca — COMMENTED** [review]
See my comment.

**Review by psolyca — DISMISSED** [review]
Good in the code.
I can not check on readthedocs, the build failed (all build), Sphinx error. It should be a readthedocs error.

**Comment by steko** [comment]
The error is caused by the old Sphinx version running on Read the Docs (1.8.5).

Perhaps ee can upgrade the dependency in this pull request even if it's unrelated? There is a number of other updates needed to the configuration but it's a larger change.

**Comment by psolyca** [comment]
Yes go on, it is a need for the doc and it is a PR for doc ^^

**Comment by steko** [comment]
It seems to build correctly on Read the Docs (we may need some adjusting of the link with GitHub in order to show the build result in pull requests).

**Comment by steko** [comment]
I managed to activate the integration with Read the Docs for pull requests.

**Comment by steko** [comment]
The built documentation is (temporarily) at https://totalopenstation.readthedocs.io/en/140-virus-alert/

@psolyca do you approve the PR?

**Review by psolyca — APPROVED** [review]
Every thing is on the documentation as expected.

**Pull request merged**